src/test/resources/com/aemtools/completion/html/attributes/htlAttributesFilterOutUnwrap3.html

```diff
@@ -0,0 +1 @@
+<div data-sly-unwrap data-sly-<caret> ></div>
```

Comment:
Not dupicate. Because caret stand after `data-sly-` and in "htlAttributesFilterOutUnwrap.html" it stand after space `<div data-sly-unwrap <caret> ></div>`. 
And these situations are different because we have additional PsiElement before <caret>